ansible/group_vars/libvirt-vms

```diff
@@ -30,3 +30,10 @@ vm_configdrive_network_device_list:
     mtu: 1500
 
 vm_hypervisor: "{{ groups['libvirt-hosts'][0] }}"
+
+vm_user_data: |
```

Comment:
We could make it undefined; I thought that having an example of how to configure it would be handy though (and seemed to follow the pattern for other variables?)

Reply:
It seems a bit more opinionated than the others (IIRC the gateway is a standard libvirt subnet). How about adding the example as a comment?

Reply:
Good idea, I've done that in the latest commit